app/controllers/reasons/BoxNumberController.scala

```diff
@@ -94,34 +94,35 @@ class BoxNumberController @Inject()(appConfig: AppConfig,
     )
   }
 
-  private[controllers] def navigateTo(value: Int) = {
+  private[controllers] def navigateTo(value: BoxNumber) = {
     value match {
-      case 22 | 62 | 63 | 66 | 67 | 68 | 99 => Redirect(controllers.reasons.routes.UnderpaymentReasonAmendmentController.onLoad(value))
+      case BoxNumber.Box22 | BoxNumber.Box62 | BoxNumber.Box63 | BoxNumber.Box66 | BoxNumber.Box67 | BoxNumber.Box68 | BoxNumber.OtherItem =>
```

Comment:
could we not import everything from BoxNumber and have the case like Box22 | Box 62 insteaad?